v2/examples/spiffe-jwt/server/main.go

```diff
@@ -0,0 +1,93 @@
+package main
+
+import (
+	"context"
+	"io"
+	"log"
+	"net/http"
+	"strings"
+
+	"github.com/spiffe/go-spiffe/v2/spiffetls/tlsconfig"
+	"github.com/spiffe/go-spiffe/v2/svid/jwtsvid"
+	"github.com/spiffe/go-spiffe/v2/workloadapi"
+)
+
+const socketPath = "unix:///tmp/agent.sock"
+
+func index(w http.ResponseWriter, r *http.Request) {
+	log.Println("Request received")
+	_, _ = io.WriteString(w, "Success!!!")
+}
+
+type authenticator struct {
+	// JWT Source used to verify token
+	jwtSource *workloadapi.JWTSource
+	// Expected audiences
+	audiences []string
+}
+
+func (a *authenticator) authenticateClient(next http.Handler) http.Handler {
+	return http.HandlerFunc(func(w http.ResponseWriter, req *http.Request) {
+		fields := strings.Fields(req.Header.Get("Authorization"))
+		if len(fields) != 2 && fields[0] != "Bearer " {
```

Comment:
This is not quite right...
```suggestion
		if len(fields) != 2 || fields[0] != "Bearer" {
```

Reply:
🤦